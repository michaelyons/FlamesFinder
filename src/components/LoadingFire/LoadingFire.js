import React from 'react';
import fire from '../LoadingFire/fire.gif';

const fireLoad = () => {
  return (
    <div>
      <img src={fire} alt="fire loading gif" className="fire-gif" />
    </div>
  );
};

export default fireLoad;
