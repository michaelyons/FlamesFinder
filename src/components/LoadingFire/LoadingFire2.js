import React from 'react';
import fire from '../LoadingFire/fire.gif';

const fireLoad2 = () => {
  return (
    <div>
      <img src={fire} alt="fire loading gif" className="fire-gif2" />
    </div>
  );
};

export default fireLoad2;
