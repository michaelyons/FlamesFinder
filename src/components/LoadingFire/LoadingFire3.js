import React from 'react';
import fire from '../LoadingFire/fire.gif';

const fireLoad3 = () => {
  return (
    <div>
      <img src={fire} alt="fire loading gif" className="fire-gif3" />
    </div>
  );
};

export default fireLoad3;
