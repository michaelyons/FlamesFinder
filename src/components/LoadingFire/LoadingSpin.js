import React from 'react';
import fire from '../LoadingFire/fire.gif';

const fireLoad = () => {
  return (
    <div>
      <img
        src={fire}
        alt="fire loading gif"
        style={{
          width: '200px',
          margin: 'auto',
          display: 'block',
          paddingTop: '12rem'
        }}
      />
    </div>
  );
};

export default fireLoad;
