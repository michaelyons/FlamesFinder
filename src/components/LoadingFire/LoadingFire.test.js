import React from 'react';
import { shallow } from 'enzyme';

import LoadingFire from './LoadingFire';

describe('LoadingFire', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<LoadingFire />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
