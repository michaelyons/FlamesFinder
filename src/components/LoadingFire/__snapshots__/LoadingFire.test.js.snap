// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingFire should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <fireLoad />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <img
        alt="fire loading gif"
        className="fire-gif"
        src="fire.gif"
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "fire loading gif",
        "className": "fire-gif",
        "src": "fire.gif",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt="fire loading gif"
          className="fire-gif"
          src="fire.gif"
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "fire loading gif",
          "className": "fire-gif",
          "src": "fire.gif",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
